### PR TITLE
[Refactor] add more log to HdfsManager

### PR DESCRIPTION
**Pull request by xiaoyong-z**
Signed-off-by: xyz <a997647204@gmail.com>

## What type of PR is this：
- [ ] bugfix
- [ ] feature
- [ ] enhancement
- [x] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 4 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/fs/hdfs/HdfsFsManager.java|0|4|00.00%|[989, 1009, 1019, 1022]|

File: fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java [hunk 1]
```diff
@@ -1015,10 +1016,10 @@ public void closeWriter(TBrokerFD fd) throws UserException {
         FSDataOutputStream fsDataOutputStream = ioStreamManager.getFsDataOutputStream(fd);
         synchronized (fsDataOutputStream) {
             try {
-                fsDataOutputStream.flush();
+                fsDataOutputStream.hsync();
```
Comment (trueeyu) [inline fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java:1019]: Why change to hsync?

Comment (xiaoyong-z) [reply fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java:1019]: To make sure the hdfs file is sync to the disk.
Especially when the disk is almost full, the hsync can detect and throw exception.

Comment (chaoyli) [reply fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java:1019]: https://github.com/StarRocks/starrocks/pull/10713 @xiaoyong-z This pull request already add the log

Comment (xiaoyong-z) [reply fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java:1019]: this pr adds log in FE, 
https://github.com/StarRocks/starrocks/pull/10713 adds log in broker

**Review by trueeyu — APPROVED** [review]

**Review by chaoyli — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by xiaoyong-z** [comment]
run starrocks_admit_test

**Pull request merged**